veggieseasons/lib/screens/home.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/widgets.dart';
+import 'package:veggieseasons/screens/favorites.dart';
+import 'package:veggieseasons/screens/list.dart';
+import 'package:veggieseasons/screens/search.dart';
+import 'package:veggieseasons/screens/settings.dart';
+
+class HomeScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoTabScaffold(
+      tabBar: CupertinoTabBar(items: [
+        BottomNavigationBarItem(
+          icon: Icon(CupertinoIcons.home),
+          title: Text('Home'),
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(CupertinoIcons.book),
+          title: Text('My Garden'),
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(CupertinoIcons.search),
+          title: Text('Search'),
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(CupertinoIcons.settings),
+          title: Text('Settings'),
+        ),
+      ]),
+      tabBuilder: (context, index) {
+        if (index == 0) {
```

Comment:
nit: switch statement?

Reply:
I actually had a switch statement in there to start, but took it out after seeing this:

https://www.dartlang.org/guides/language/language-tour#switch-and-case

Your Dart expertise trumps mine, though. Should I change it back?